src/ontology/mondo-edit.obo

```diff
@@ -313672,8 +313654,8 @@ xref: ICD10CM:Q83.0 {source="Orphanet:180188/e", source="Orphanet:180188/specifi
 xref: OMIMPS:113700 {source="MONDO:equivalentTo"}
 xref: Orphanet:180188 {source="MONDO:equivalentTo"}
 xref: UMLS:C0432357 {source="MONDO:notFoundInDiseaseSubset", source="Orphanet:180188"}
-is_a: MONDO:0002263 {source="Orphanet:180188"} ! female reproductive system disorder
```

Comment:
was this edit from curator and therefore intended?

Reply:
yes, and you can see below in line 313658 that I excluded the superclass